### PR TITLE
Add support for more audio types

### PR DESCRIPTION
**Pull request by rotvel**
TL;DR fix directplay of mp3 and m4a(aac) and add support for directplay of mka(opus) and ogg(vorbis). 

The line `builder.append("&Container=flac")` plus the transcoding information tells Jellyfin that Gelli only supports directplay of flac files. Everything else - including mp3 and aac! - will be trancoded to mp3 or aac.

This pr adds support for the most common audio formats. I don't think I missed any important ones, no one streams wav, right? :) 

All formats have been tested against Jellyfin 10.5.5 and the latest master, using Gelli 1.0.2 on my phone. And they all played perfectly without any trancoding.

(the /.vs addition to .gitignore is just junk from Visual Studio)

### CODE REVIEW

**Comment by rotvel** [comment]
For completeness sake: the x|y format means container|codec. 

Eg: m4a|aac = for the .m4a container I'm only interested in aac encoded files.

**Comment by dkanada** [comment]
Any chance you want to make this a multi-select preference at the bottom of the settings? Just in case a user runs into direct play issues they could disable certain codecs and such. I haven't tested this endpoint without the container parameter, wonder if that would force direct play...been trying to figure out a nice method to handle that.

I eventually want a toggle button somewhere to switch between auto, transcode, and direct but I'm not sure we have an official method to force one or the other, so I'll have to rely on client side tricks.

**Comment by rotvel** [comment]
I tried testing the 10.5.5 endpoint with no Container parameter and it worked as you suggested by transcoding everything.

**Comment by rotvel** [comment]
I'm looking into creating a multi-select preference for the directplay codecs as you requested. Hacking away in Android Studio right now. 

I can confirm that the 10.6.0 endpoint works similar to the 10.5.5 one: no Container parameter forces transcoding

**Comment by rotvel** [comment]
Done. Android Studio changed some build files. These are in a separate commit as I'm not sure if they should go in.

**Comment by dkanada** [comment]
> Android Studio changed some build files.

We can't update those values until scoped storage is implemented for newer versions of Android or several bugs will appear with the caching system. I can revert that commit or something when I merge this though.

**Comment by dkanada** [comment]
Did some linting and used the upstream name `Direct Play` for all string and code instances. Not sure if the names will change at some point (I think they're a bit unclear personally) but it's better to match them exactly for now.

**Comment by rotvel** [comment]
Ok. Thanks for fixing that.

**Comment by grravity** [comment]
Is there anything that can be done about the lack of support for 24bit audio? 
Seems like it is a Jellyfin issue not a client issue, so I didn't want to open a new issue just for this.
